Pipfile

```diff
@@ -29,6 +29,7 @@ twine = "*"
 
 [packages]
 integreat-cms = {editable = true, path = "."}
+django-db-mutex = "*"
```

Comment:
```suggestion
```
Functional dependencies need to be added to the [setup.cfg](https://github.com/digitalfabrik/integreat-cms/blob/534f226f80618be350bb10c1e6d7c7448147db66/setup.cfg#L43) (At least until #2027 is merged...)